WebRoot/js/zimbraAdmin/servers/view/ZaServerXFormView.js

```diff
@@ -669,7 +677,7 @@ ZaServerXFormView.BIND_IP_TAB_RIGHTS = [];
 ZaServerXFormView.myXFormModifier = function(xFormObject, entry) {	
 	var headerList = new Array();
 	headerList[0] = new ZaListHeaderItem(ZaServer.A_VolumeName, ZaMsg.VM_VolumeName, null, "100px", false, null, false, true);
-	headerList[1] = new ZaListHeaderItem(ZaServer.A_VolumeRootPath, ZaMsg.VM_VolumeRootPath, null,"200px", false, null, false, true);
+	headerList[1] = new ZaListHeaderItem(ZaServer.A_VolumeRootPath, "Volume Path or S3 Prefix", null,"200px", false, null, false, true);
```

Comment:
add `"Volume Path or S3 Prefix"` to properties files for translation

Reply:
done